api/app/signals/apps/signals/migrations/0155_alter_attachment_options.py

```diff
@@ -0,0 +1,25 @@
+# SPDX-License-Identifier: MPL-2.0
+# Copyright (C) 2022 Vereniging van Nederlandse Gemeenten
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('signals', '0154_statusmessagetemplate_is_active'),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name='attachment',
+            options={
+                'ordering': ('created_at',),
+                'permissions': [
```

Comment:
See comment about permissions in the Attachment model.

Reply:
All new custom permission names start with `sia_`